sdk/communication/AzureCommunicationCommon/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 1.1.0 (upcoming)
+### New Features
+- Introduce new `CommunicationIdentifierHelper` to create `CommunicationIdentifier` based on rawIds
```

Comment:
Would be nice to have a more detailed explanation of the change, similar to https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/communication/communication-common/CHANGELOG.md#features-added